src/ark/utils/plot_utils.py

```diff
@@ -293,17 +312,16 @@ def plot_pixel_cell_cluster_overlay(img_xr, fovs, cluster_id_to_name_path, metac
             cax=cax,
             orientation='vertical'
         )
-        cbar.ax.set_yticklabels(['Empty'] + list(
+        cbar.ax.set_yticklabels(
             mcc.metacluster_id_to_name[f'{cluster_type}_meta_cluster_rename'].values)
```

Comment:
Is "Empty" now included in `metacluster_id_to_name`?

Reply:
Yup